### PR TITLE
feat(tikv/migration): add ut and it ci

### PR DESCRIPTION
**Pull request by purelind**
* add unit-test and integration-test ci pipeline for tikv/migration

Already test those pipelines on staging env
1. https://prow-ee.pingcap.net/jenkins/job/tikv/job/migration/job/pull_unit_test/7/
2. https://prow-ee.pingcap.net/jenkins/job/tikv/job/migration/job/pull_integration_test/6/

### CODE REVIEW

File: pipelines/tikv/migration/latest/pull_unit_test.groovy [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                                ]
+                            )
+                        }
+                    }
+                }
+            }
+        }
+        stage('Unit Tests') {             
+            stages {
+                stage("Test") {
```
Comment (wuhuizuo) [inline pipelines/tikv/migration/latest/pull_unit_test.groovy:73]: too many nested levels

File: pipelines/tikv/migration/latest/pull_unit_test.groovy [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    }
+                    post{
+                        always {
+                            junit(testResults: "**/cdc-junit-report.xml")
+                        }
+                    }
+                }
+            }         
+        }
+    }
```
Comment (wuhuizuo) [inline pipelines/tikv/migration/latest/pull_unit_test.groovy:92]: should it report result in post of pipeline?

**Review by zeminzhou — APPROVED** [review]
LGTM～, thanks again!

File: pipelines/tikv/migration/latest/pull_integration_test.groovy [hunk 3]
```diff
@@ -0,0 +1,120 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+// should triggerd for master and latest release branches
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-tidb"
```
[the remark this answers is not in the document]
Comment (purelind) [reply pipelines/tikv/migration/latest/pull_integration_test.groovy:6]: you suggest not distinguishing namespace on ksyun just a few weeks ago

File: pipelines/tikv/migration/latest/pull_unit_test.groovy (hunk 2 above)
Comment (purelind) [reply pipelines/tikv/migration/latest/pull_unit_test.groovy:92]: No need now

File: pipelines/tikv/migration/latest/pull_unit_test.groovy [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                        }
+                    }
+                }
+            }
+        }
+        stage('Unit Tests') {             
+            stages {
+                stage("Test") {
+                    options { timeout(time: 25, unit: 'MINUTES') }
+                    environment {TIKV_MIGRATION_CODECOV_TOKEN = credentials('codecov-token-tikv-migration')}
```
[the remark this answers is not in the document]
Comment (purelind) [reply pipelines/tikv/migration/latest/pull_unit_test.groovy:75]: will format this later

**Pull request merged**